packages/backend/tsconfig.json

```diff
@@ -2,8 +2,6 @@
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
     "baseUrl": ".",
-    "declaration": true,
```

Comment:
When touching the existing `tsconfig.json` I removed properties that are defined in `tsconfig.declarations.json`, just cleaning things up